src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,24 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTab, onTabClick }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            key={tab.id}
+            className={tab.id === selectedTab.id ? 'is-active'
+              : ''}
+            data-cy="Tab"
+          >
+            <a
+              href={`#${tab.id}`}
+              onClick={() => onTabClick(tab)}
```

Comment:
<img width="854" alt="image" src="https://github.com/mate-academy/react_tabs-js/assets/70220667/906b4ce3-d260-4b38-b750-0a2b11fd1c0e">
 you forgot to implement this option in the parentheses. add the check if tab.id === selectedTab.id then do not call the onTabClick